lib/policies_finder_publisher.rb

```diff
@@ -67,10 +67,31 @@ def details
         document_type: "policy"
       },
       show_summaries: false,
-      facets: [],
+      facets: facets,
     }
   end
 
+  def facets
+    [
+      {
+        key: "public_timestamp",
+        short_name: "Updated",
+        name: "Published",
```

Comment:
It's 'published' on the individual policy finders (
https://github.com/alphagov/policy-publisher/blob/master/app/presenters/content_item_presenter.rb#L141-L148),
so was going for consistency on that, easier enough to change if someone
agrees.

On 6 May 2015 at 17:22, Tommy Palmer <notifications@github.com> wrote:

> In lib/policies_finder_publisher.rb
> <https://github.com/alphagov/policy-publisher/pull/60#discussion_r29774946>
> :
>
> >      }
> >    end
> >
> > +  def facets
> > +    [
> > +      {
> > +        key: "public_timestamp",
> > +        short_name: "Updated",
> > +        name: "Published",
>
> Should this be "Last Updated" so it matches with the use of "Updated"?
> @bmwelby <https://github.com/bmwelby> / @futurefabric
> <https://github.com/futurefabric>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/alphagov/policy-publisher/pull/60/files#r29774946>.
>



-- 
*David Singleton*

Cabinet Office, Government Digital Service
Developer, GOV.UK
david.singleton@digital.cabinet-office.gov.uk